### DISCUSSION TITLE
v5.0.0

### DISCUSSION THREAD

**Discussion by SiAdcock** [guardian/source#1281] [General]
# About this discussion

This discussion contains proposals for v5.0.0 features of Source. It remains an open discussion and nothing is set in stone (until it is).

We will keep this post up-to-date with the full list of candidates and their status.

If you have ideas, please add one post per idea, so we can have a permalink to each idea.

# Candidate changes

- Make `label` props in `CheckboxGroup`, `ChoiceCardGroup` and `RadioGroup` required 
- Make `label` props in `Checkbox` and `Radio` required. Introduce an optional `hideLabel` prop
- Remove dependency on emotion

**Comment by sndrs** [comment]
Remove dependency on emotion

**Comment by SiAdcock** [comment]
Required labels:

- if label is to be hidden, we can still display the label using the `title` attribute
